docs/extensions/readme.md

```diff
@@ -1173,3 +1175,59 @@ An object with an optional property.
   }
 }
 ```
+
+## x-ms-header-collection-prefix
+Use `x-ms-header-collection-prefix` to deal with prefixes for headers. Has different behavior if it refers to a request header or a response header:
```

Comment:
I would rephrase this a little -  it's about handling collections of arbitrary headers by distinguishing them using a specified prefix.